view/src/components/farms.js

```diff
@@ -755,11 +698,11 @@ class Farms extends Component {
                             Point of Contact:
                           </Typography>
                           <Typography variant="body2" component="p">
-                            Name: {farm.contactName}
+                            Name: {farm.contacts[0]["contactName"]}
                             <br />
-                            Phone: {farm.contactPhone}
+                            Phone: {farm.contacts[0]["contactPhone"]}
                             <br />
-                            Email: {farm.contactEmail}
+                            Email: {farm.contacts[0]["contactEmail"]}
```

Comment:
are we guaranteed that contacts[0] will be defined? do we only want the first contact?